pom.xml

```diff
@@ -4,13 +4,23 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
-    <artifactId>Automation_w1</artifactId>
+    <groupId>com.epam.automation-menthoring</groupId>
+    <artifactId>Calculator</artifactId>
```

Comment:
package name, artifact id always lowercase, please change it accordingly

Reply:
Done